docs: update resty example (#160)
jarcoal/httpmock: README.md

import (
  // ...
  "github.com/jarcoal/httpmock"
)

type Article struct {
	Status struct {
		Message string `json:"message"`
		Code    int    `json:"code"`
	} `json:"status"`
}

var _ = Describe("Articles", func() {
  It("returns a list of articles", func() {
    fixture := `{"status":{"message": "Your message", "code": 200}}`

>     // have to use JsonResponder to get a content-type header application/json

Reviewer: ```suggestion
    // have to use NewJsonResponder to get an application/json content-type
```

>     // alternately, create a go object instead of using json.RawMessage

Reviewer: ```suggestion
    // alternatively, create a go object instead of using json.RawMessage
```
